Dockerfile

```diff
@@ -34,6 +34,6 @@ LABEL vendor="Dell Inc." \
     name="csi-powermax" \
     summary="CSI Driver for Dell EMC PowerMax" \
     description="CSI Driver for provisioning persistent storage from Dell EMC PowerMax" \
-    version="2.12.0" \
+    version="2.13.0" \
```

Comment:
How about related changes for csireverseproxy?
_csireverseproxy/Dockerfile_ - I see this in previous version PR.

Reply:
done